repl/src/main/scala/ammonite/repl/interp/Interpreter.scala

```diff
@@ -41,7 +47,34 @@ class Interpreter(shellPrompt0: Ref[String],
         printer
       )
     } finally Thread.currentThread().setContextClassLoader(oldClassloader)
-  } yield out
+  }
+
+  def processScript(code: String): Unit = {
+    val blocks = Parsers.splitScript(code).map(preprocess(_, eval.getCurrentLine))
+    val errors = blocks.collect{ case Res.Failure(err) => err }
+    if(!errors.isEmpty) 
```

Comment:
Be consistent whether or not you use curlies for single-line expresisons in an if-else. I don't care which way you pick, but it should be the same for both clauses =P